index.md

```diff
@@ -29,9 +29,9 @@ Burn-P3 utilized a single deterministic fire growth model called [Prometheus](ht
 
 The **BurnP3+ SyncroSim Package** requires the SyncroSim software, [version 3.0.9](https://syncrosim.com/download/){:target="_blank"}. <br>
 
-If using the Cell2Fire fire growth model, you will also need to install the [BurnP3+Cell2Fire](https://github.com/BurnP3/BurnP3PlusCell2Fire){:target="_blank"}. <br>
+If using the Cell2Fire fire growth model, you will also need to install the [Cell2Fire](https://github.com/BurnP3/BurnP3PlusCell2Fire){:target="_blank"}. <br>
```

Comment:
This is what I meant by my comment:

> can you also change "the BurnP3+Prometheus" and "the BurnP3+Cell2Fire" to just the package names

remove the "the" in front of the package name. But keep the "BurnP3+" in front of "Cell2Fire"